src/psyclone/domain/gocean/gocean_constants.py

```diff
@@ -32,46 +32,65 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Author: J. Henrichs, Bureau of Meteorology
+# Modified: R. W. Ford, STFC Daresbury Lab
 
 '''
 This module provides a class with all GOcean related constants.
 '''
 
 # Imports
-from __future__ import print_function, absolute_import
+from psyclone.configuration import Config
 
 
 # pylint: disable=too-few-public-methods
 class GOceanConstants(object):
-    '''This class stores all GOcean constants.
-    It stores all values in class variables (to avoid re-evaluating them).
-    At this stage it only contains the variables that might be used in
-    psyGen.
+    '''This class stores all GOcean constants. It stores all values in
+    class variables (to avoid re-evaluating them).
 
     '''
     HAS_BEEN_INITIALISED = False
 
+    @staticmethod
+    def get_valid_access_types():
+        '''Return the valid access types for the GOcean API. Reads the values
+            from the config file the first time the method is called.
```

Comment:
Please could you tidy the indentation of this doc string.

Reply:
OK.